tests/run_tests.py

```
@@ -6,79 +6,83 @@
-    
+
     # Check API keys
     api_keys = {
-        'TOGETHER_API_KEY': 'Together AI tests',
-        'OLLAMA_AVAILABLE': 'Ollama tests (check if Ollama is running)'
+        "TOGETHER_API_KEY": "Together AI tests",
+        "OLLAMA_AVAILABLE": "Ollama tests (check if Ollama is running)",
     }
-    
+
     print(f"\nAPI Key Status:")
```

Ruff (0.14.6)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)